localgov_directories.module

```diff
@@ -467,3 +467,21 @@ function localgov_directories_remove_block_from_content_type_sidebar(string $blo
 
   return TRUE;
 }
+
+/**
+ * Implements hook_search_api_index_items_alter().
+ *
+ * Populate localgov_directory_title_sort if empty.
+ */
+function localgov_directories_search_api_index_items_alter(IndexInterface $index, array &$items) {
+  foreach ($items as $item) {
+    if ($field = $item->getField('localgov_directory_title_sort')) {
+      $sort_value = $field->getValues();
+      if (empty($sort_value) || empty($sort_value[0])) {
+        // If the field is empty use the item title.
+        $sort_value = [$item->getOriginalObject()->getEntity()->label()];
```

Comment:
Any chance you can please pass the ```$sort_value``` through ```trim()```?  I have found a Directory venue page in our site whose title has a space at the beginning.  Possibly entered by mistake!  It is sitting at the top of the list as a result.

Reply:
So I was thinking about this.

My immediate reaction was we should then trim the field itself, rather than the default value entered into the field,
*but* then I thought if you are entering a string for sorting that's really the value it should be sorted on with exactly what was entered,
so then trim()ing the automated output starts to make sense, so I'll add that.